### ISSUE TITLE
Doesn't appear in menu

### ISSUE THREAD

**Issue by jacksongoode** [bjarosze/gnome-bluetooth-quick-connect#52]
This could be related to some GNOME update maybe?

**Comment by bdaase** [comment]
Also, the auto power off timeout kicks in even though devices are connected.
A quick grep though the code indicates that the connected devices doesn't get updated properly anymore and stays 0 all the time.

**Comment by zheneknew** [comment]
Yes, it doesn't appear in menu in fedora 35.
It worked fine in fedora 34.

**Comment by jacksongoode** [comment]
Looks like version 27 should address this still seems to have the same behavior.

**Comment by jacksongoode** [comment]
It seems that when Bluetooth is turned off (automatically) the menu foldout will disappear.

**Comment by Jegah** [comment]
Yeah by default, gnome only show the bluetooth menu entry when bluetooth is on. You can set it to always be there with :

`gsettings set org.gnome.shell had-bluetooth-devices-setup true`

After a restart it should be there even if bluetooth is off. That should probably be the default nowdays.
I got that solution from an similar issue on a new extension called [quick-settings-extension](https://gitlab.gnome.org/fmuellner/quick-settings-extension/-/issues/1)
